### ISSUE TITLE
libtnzext.so and superlu.a (build failed)

### ISSUE THREAD

**Issue by VirgilioVasconc** [ideasman42/opentoonz#18]
Hey guys. After the new commits I cannot build OpenToonz again. It was ok before the last merge with the main branch. The error happens on Ubuntu 16.04 64 bit.
```
[ 12%] Linking CXX shared library libtnzext.so
/usr/bin/ld: /usr/lib/gcc/x86_64-linux-gnu/6/../../../x86_64-linux-gnu/libsuperlu.a(util.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/lib/gcc/x86_64-linux-gnu/6/../../../x86_64-linux-gnu/libsuperlu.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
tnzext/CMakeFiles/tnzext.dir/build.make:838: recipe for target 'tnzext/libtnzext.so' failed
make[2]: *** [tnzext/libtnzext.so] Error 1
CMakeFiles/Makefile2:229: recipe for target 'tnzext/CMakeFiles/tnzext.dir/all' failed
make[1]: *** [tnzext/CMakeFiles/tnzext.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```
It tells me to try and recompile with the -fPIC parameter. I have tried to ```export CFLAGS=-fPIC``` but I get the same error.

**Comment by andreygursky** [comment]
I believe this is the same story like with libtiff: https://github.com/ideasman42/opentoonz/issues/13, but now with libsuperlu. Just revert https://github.com/ideasman42/opentoonz/commit/aeb132d360174e7adc070e6bc308ca15454cafaf.

**Comment by VirgilioVasconc** [comment]
That was it! I have changed the file toonz/cmake/FindSuperLU.cmake, reverting libsuperlu.a to libsuperlu.so and it has passed that point in compilation. Thank you, Andrey!

Unfortunately, the compilation has failed afterwards: ``` toonz/sources/image/png/tiio_png.cpp:112:39: error: ‘png_set_longjmp_fn’ was not declared in this scope
                        sizeof(jmp_buf)); /* ignore all fatal errors */
```
I have got libpng12-dev installed on my machine, and it was ok for compiling it. I'll try digging it further to see what I get. :)
